query.go

```diff
@@ -135,8 +136,6 @@ processFollowUp:
 }
 
 func (dht *IpfsDHT) runQuery(ctx context.Context, target string, queryFn queryFn, stopFn stopFn) (*lookupWithFollowupResult, error) {
-	queryCtx, cancelQuery := context.WithCancel(ctx)
-
```

Comment:
What's the rationale behind removing this ?

Reply:
We weren't using it for anything as far as I can tell